pkg/cmd/service/service.go

```diff
@@ -147,6 +148,29 @@ type Config struct {
 	Profiler   ProfilerConfig
 }
 
+func DefaultConfigWithRandomPorts() (*Config, error) {
```

Comment:
Is this more of a test utility, rather than something that should be in a non-test file?

Reply:
It could still be useful outside of the scope of tests. If you want to pass config to OpenFGA that uses random ports then this works for you. That's why I put it there.

Reply:
👍